makefile

```diff
@@ -131,32 +126,14 @@ run: kill build
 		do echo "waiting for postgres, localstack bootup"; \
 		sleep 0.5; \
 		done
-	sed -i 's/dockerdns/$(DOCKER_DNS)/g' ./.env
 	make superposition -e DOCKER_DNS=$(DOCKER_DNS)
 
-ci-test: cleanup ci-setup
```

Comment:
why remove cleanup from here?

Reply:
We don't need docker cleanup anymore, github actions automatically cleans up containers